src/libs/antares/benchmarking/file_content.cpp

```diff
@@ -0,0 +1,32 @@
+#include "file_content.h"
+
+using namespace std;
+
+namespace Benchmarking
+{
+	FileContent::iterator FileContent::firstSection()
+	{
+		return sections_.begin();
+	}
+
+	FileContent::iterator FileContent::endSections()
+	{
+		return sections_.end();
+	}
+
+	void FileContent::addItemToSection(string section, string key, int value)
+	{
+		sections_[section][key] = to_string(value);
+	}
+
+	void FileContent::addItemToSection(string section, string key, string value)
+	{
+		sections_[section][key] = value;
+	}
+
+	void FileContent::addDurationItem(std::string name, unsigned int duration, int nbCalls)
```

Comment:
1. Pass strings by const-ref
2. Use `string` version in the `int` implementation.

```suggestion
	void FileContent::addItemToSection(const std::string& section, const std::string& key, int value)
	{
		this->addItemToSection(section, key, std::to_string(value));
	}

	void FileContent::addItemToSection(const std::string& section, const std::string& key, const std::string& value)
	{
		sections_[section][key] = value;
	}

	void FileContent::addDurationItem(const std::string& name, unsigned int duration, int nbCalls)
```

Reply:
Done